creditrisk_poc/api_doc/ApiDoc.jsonld

```diff
@@ -611,6 +717,103 @@
                     "required": null,
                     "writeable": false
                 },
+                {
+                    "hydra:description": "The Collateral Class",
```

Comment:
there shouldn't be here an `@id` or a `@type` that points to the `Collateral` class in the original ontology?
cc @chrizandr

Reply:
@Mec-iS Complete descriptions are not added as we are more focused on functionality as of now.